src/app/components/vertical-tabs/index.html

```diff
@@ -20,16 +20,44 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Vertical tab properties">
+    <sky-demo-page-property
+      propertyName="tabHeading"
+    >
+    Specifies the tab header.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="active"
       isOptional="true"
     >
-    Indicates whether the tab is active when the tabset loads.
+    Indicates whether the tab is active when the tabset loads. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="tabHeading"
+      propertyName="ariaControls"
+      isOptional="true"
     >
-    Specifies the tab header.
+    Specifies an ID to identify the element that contains the content the vertical tab exposes. This property supports <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure">accessibility rules for disclosures</a>. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="ariaInvalid"
```

Comment:
I just noticed that the UX guidelines for vertical pages say "Do not use vertical tabs on a form. Use the sectioned form component instead." ... So the descriptions of `ariaInvalid` and `ariaRequired` contradict the UX guidelines. I think Conor provided the descriptions in https://github.com/blackbaud/skyux2/issues/1840 because he did the dev work, but I think we should reach out to Todd about this. It seems like we probably need to include a recommendation in the property descriptions to NOT use vertical tabs in forms, but maybe Todd will have a different recommendation.

Reply:
I'll reach out to Todd about this one

Reply:
Chatted with Alex, and he suggested moving `ariaInvalid` and `ariaRequired` to the Sectioned form component docs to stay consistent with design guidelines.